src/components/ExampleComponents.tsx

```diff
@@ -1,39 +1,236 @@
+  render(): JSX.Element {
+    return <OrigModal {...this.props} />;
+  }
+}
+addDisplayNames(Modal, OrigModal, 'Modal');
+
+export class ModalContent extends Component<ModalContentProps> {
+  render(): JSX.Element {
+    return <OrigModalContent {...this.props} />;
+  }
+}
+addDisplayNames(ModalContent, OrigModalContent, 'ModalContent');
+
+export class ModalTitle extends Component<ModalTitleProps> {
+  render(): JSX.Element {
+    return <OrigModalTitle {...this.props} />;
+  }
+}
+addDisplayNames(ModalTitle, OrigModalTitle, 'ModalTitle');
+
+export class ModalFooter extends Component<ModalFooterProps> {
+  render(): JSX.Element {
+    return <OrigModalFooter {...this.props} />;
+  }
+}
+addDisplayNames(ModalFooter, OrigModalFooter, 'ModalFooter');
+
+export const MultiSelect = function <T extends MultiSelectData>(
+  props: MultiSelectProps<T>,
+): JSX.Element {
+  return <OrigMultiSelect {...props} />;
+};
+addDisplayNames(MultiSelect, OrigMultiSelect, 'MultiSelect');
+
+export class Paragraph extends OrigParagraph {}
+addDisplayNames(Paragraph, OrigParagraph, 'Paragraph');
+
+export const RadioButton = (props: RadioButtonProps): JSX.Element => (
+  <OrigRadioButton {...props} />
+);
+addDisplayNames(RadioButton, OrigRadioButton, 'RadioButton');
+
+export const RadioButtonGroup = (props: RadioButtonGroupProps): JSX.Element => (
+  <OrigRadioButtonGroup {...props} />
+);
+addDisplayNames(RadioButtonGroup, OrigRadioButtonGroup, 'RadioButtonGroup');
 
 export class SearchInput extends OrigSearchInput {}
 addDisplayNames(SearchInput, OrigSearchInput, 'SearchInput');
 
-export class Icon extends OrigIcon {}
-addDisplayNames(Icon, OrigIcon, 'Icon');
-
 export class StaticIcon extends OrigStaticIcon {}
 addDisplayNames(StaticIcon, OrigStaticIcon, 'StaticIcon');
 
-export class Block extends OrigBlock {}
-addDisplayNames(Block, OrigBlock, 'Block');
+export class Text extends OrigText {}
+addDisplayNames(Text, OrigText, 'Text');
 
-export class LanguageMenu extends OrigLanguageMenu {}
-addDisplayNames(LanguageMenu, OrigLanguageMenu, 'LanguageMenu');
+export const Textarea = (props: TextareaProps): JSX.Element => (
+  <OrigTextarea {...props} />
+);
+addDisplayNames(Textarea, OrigTextarea, 'Textarea');
 
-export class LanguageMenuItem extends OrigLanguageMenuItem {}
-addDisplayNames(LanguageMenuItem, OrigLanguageMenuItem, 'LanguageMenuItem');
+export const TextInput = (props: TextInputProps): JSX.Element => (
+  <OrigTextInput {...props} />
+);
+addDisplayNames(TextInput, OrigTextInput, 'TextInput');
+
+export const ToggleButton = (props: ToggleButtonProps): JSX.Element => (
+  <OrigToggleButton {...props} />
+);
+addDisplayNames(ToggleButton, OrigToggleButton, 'ToggleButton');
+
+export const ToggleInput = (props: ToggleInputProps): JSX.Element => (
+  <OrigToggleInput {...props} />
+);
+
+addDisplayNames(ToggleInput, OrigToggleInput, 'ToggleInput');
+
+export class VisuallyHidden extends Component<any> {
```

Comment:
I assume that this is `any` because in the current version those **VisuallyHiddenProps** are not available in comps.

Let's try to remember to change this when those props are available.

Reply:
Yes, I found this to be a better solution than copying the types.